Google.Api.Generator/Generation/ServiceDetails.cs

```diff
@@ -41,59 +41,47 @@ public ServiceDetails(ProtoCatalog catalog, string ns, ServiceDescriptor desc)
             desc.CustomOptions.TryGetMessage<OAuth>(ProtoConsts.ServiceOption.OAuth, out var oauth);
             DefaultScopes = oauth?.Scopes ?? Enumerable.Empty<string>();
             Methods = desc.Methods.Select(x => MethodDetails.Create(this, x)).ToList();
+            SnippetsTyp = Typ.Manual(ns, $"Generated{desc.Name}Snippets");
+            SnippetsClientName = $"{desc.Name.ToLowerCamelCase()}Client";
         }
 
         public ProtoCatalog Catalog { get; }
         public string Namespace { get; }
 
-        /// <summary>
-        /// The name of this service to be used in documetation.
-        /// </summary>
+        /// <summary>The name of this service to be used in documetation.</summary>
```

Comment:
Nit: documetation => documentation
(Fix to existing issue)

Reply:
Done